x-pack/plugin/eql/qa/common/src/main/java/org/elasticsearch/test/eql/EqlSpecLoader.java

```diff
@@ -55,6 +55,14 @@ private static String getTrimmedString(TomlTable table, String key) {
             spec.note(getTrimmedString(table, "note"));
             spec.description(getTrimmedString(table, "description"));
 
+            // default these values to false
+            spec.caseSensitiveOnly(Boolean.TRUE.equals(table.getBoolean("case_sensitive")));
```

Comment:
I find the "parameter" from the TOML tests a bit confusing. At Java level it's `caseSensitiveOnly` and in tests is `case_sensitive`. Imo, it's an important difference between something being "case sensitive ONLY" and "case sensitive".

A test that is "case sensitive" can, also, be "case insensitive", meaning a test that doesn't care about case sensitivity at all. For example, a test where no string functions are involved: this one could run in whatever configuration. My suggestion is to have in `.toml` files the `only` bit added to parameters to make a bit more clear.

Reply:
in 254b5c3, I removed the implicit behavior for and the concept of "case (in)sensitive only".
now every test requires `case_sensitive=true` and `case_insensitive=true` if it should be tested with both